frontend/packages/@depmap/dataset-manager/src/components/index.tsx

```diff
@@ -115,7 +118,17 @@ export default function Datasets() {
   useEffect(() => {
     (async () => {
       try {
-        const currentDatasets = await dapi.getBreadboxDatasets();
+        let currentDatasets = await dapi.getBreadboxDatasets();
+
+        if (!isAdvancedMode) {
```

Comment:
It appears get groups is only called here. Can't you skip creating the callback and just call `dapi.getGroups(!isAdvancedMode)` here?

The way that the callback was a function of his advanced mode I was expecting it to be used in advanced mode and non-advanced mode, but it looks like the only way this ever gets used is if isAdvancedMode is set to false. 

So I think this all works it's just it would be easier to follow the logic if that was explicit. Having the call back created as a layer of indirection where I have to go back up to the top to see how it was defined.